synapseclient/core/cache.py

```diff
@@ -281,22 +280,34 @@ def _cache_dirs(self):
                     if os.path.isdir(path2) and re.match('\\d+', item2):
                         yield path2
 
-    def purge(self, before_date, dry_run=False):
+    def purge(self, before_date=None, after_date=None, dry_run=False):
         """
```

Comment:
Can you add parameter description to the docstring here:

e.g.

        :param before_date: Files last updated before this date will be deleted

etc.

Reply:
Could you add the docstrings for these parameters? both the new after_date and the existing before_date and dry_run parameters?